src/dbnode/encoding/m3tsz/timestamp_encoder.go

```diff
@@ -49,13 +50,16 @@ type TimestampEncoder struct {
 	hasWrittenFirst bool
 }
 
+var emptyAnnotationChecksum = xxhash.Sum64(nil)
```

Comment:
0 is probably fine as a default

Reply:
One of the tests showed that I actually need this value somewhere where the checksum is computed without checking for the length.